src/test/java/com/wikia/webdriver/testcases/mobilewikitests/curatedcontenttests/EditorTests.java

```diff
@@ -42,7 +42,7 @@ public void beforeMethod() {
   @Test(groups = "MercuryCuratedEditorTest_001")
   @RelatedIssue(issueID = "XF-241")
   public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
-    Boolean result = new ArticlePage().open(MercurySubpages.ECC_MAIN_PAGE).getCuratedMainPage()
+    Boolean result = new ArticlePage().open(MobileSubpages.ECC_MAIN_PAGE).getCuratedMainPage()
         .isCuratedElementVisible(FEATURED_CONTENT_SELECTOR);
 
     Log.info(String.format("Curated content is visible: %s", result));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Curated content is visible: %s" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)